usb-spec.h

```diff
@@ -1385,8 +1384,7 @@ static const struct genericstrtable hutus[] =
 /* dialects are represented as:
  *	langid + (dialectid << 10)
  */
-static const struct genericstrtable langids[] =
-{
+static const struct genericstrtable langids[] = {
```

Comment:
Formatting change is fine, but again, submit code style formatting changes different from spelling fixes.